# Map Parameters
VertexCount: 5
size_x: 3
size_y: 3

# Algorithm Parameters
DistancePower: 1.0 # Increase to avoid long trips.
PheromonePower: 1.0 # Increase to trust other ants.
EvaporationPower: 0.2 # Increase to remove older paths.
RewardPower: 0.0 # Increase to favor newest paths.